frontend/tool/webpack.config.js

```diff
@@ -92,10 +92,10 @@ const config = {
                 use: getLoaders(isDev, 'stylus')
             },
             {
-                test: /\.(gif|png|jpe?g)$/i,
+                test: /\.(gif|png|jpe?g|svg)$/i,
                 loader: 'file-loader',
                 options: {
-                    name: 'images/[name].[hash].[ext]'
+                    name: 'assets/[name].[hash].[ext]'
```

Comment:
Do we have any images left in the images directory?

Reply:
nope